### PR TITLE
New input textbox

### PR DESCRIPTION
**Pull request by marilynechan**
I also updated placeholder colors everywhere, and removed the annoying spaces at the beginning of every component source in the docs.

### CODE REVIEW

File: scss/lib/_mixins.scss [hunk 1]
```diff
@@ -18,3 +18,37 @@
     height: 0;
   }
 }
+
+@mixin placeholder($text-color: $medium-grey, $font-family: $base-font-family) {
```
Comment (wfortin) [inline scss/lib/_mixins.scss]: Ha! Great.

File: scss/controls/inputs.scss [hunk 2]
```diff
@@ -73,3 +42,51 @@ input[type='text'], textarea {
...
+      & + .textbox-label {
+        top: -1 * $label-top;
+
+        color: $medium-blue;
+        font-size: 13px;
+        font-weight: bold;
+      }
+    }
+  }
+  .textbox-label {
```
Comment (wfortin) [inline scss/controls/inputs.scss]: Add a blank line before.

File: scss/lib/_mixins.scss (hunk 1 above)
Comment (wfortin) [inline scss/lib/_mixins.scss]: Looks like this is now supported by Autoprefixer 
https://github.com/postcss/autoprefixer/issues/44

So you could only use `::placeholder  { }`

**Comment by wfortin** [comment]
Please fix the autoprefixer issue, other than that :shipit: !!

File: scss/controls/inputs.scss [hunk 3]
```diff
@@ -73,3 +42,51 @@ input[type='text'], textarea {
...
 }
+
+.textbox {
+  position: relative;
+
+  margin-top: $label-top * 2;
+
+  font-size: 15px;
+
+  $label-top: 15px;
```
Comment (marilynechan) [inline scss/controls/inputs.scss]: That's weird, Sass formatter has moved this variable below where one of its usage. Moved it out of `.textbox` so it won't be moved wrongly...

File: scss/controls/inputs.scss (hunk 2 above)
Comment (marilynechan) [inline scss/controls/inputs.scss]: Done!

File: scss/lib/_mixins.scss (hunk 1 above)
Comment (marilynechan) [inline scss/lib/_mixins.scss]: Nice! Done!

**Comment by Unetitebiere** [comment]
nice! :shipit:

**Comment by wfortin** [comment]
:ship: it

**Pull request merged**